README.md

```diff
@@ -37,18 +37,23 @@ Firstly, ensure you have docker running (if you are running with a slave, ensure
 
 Setup a build of any type - with a `CloudBees Docker Build and Publish` build step.
 You can use the example under [`src/test/example`](https://github.com/jenkinsci/docker-build-publish-plugin/tree/master/src/test/example) to build a very simple busybox based image,
-and push it to `jenkinsci/docker-build-and-publish-example`.
+and push it to `acme/test`.
 
 ![build instructions](https://raw.githubusercontent.com/jenkinsci/docker-build-publish-plugin/master/build-config.png)
 
-The usual Docker build caching mechanism applies - and you can choose to publish, or not, the resultant image, configured under Advanced options.
+A remote Docker server can be configured providing its Docker server URI and creating a `Docker Server Certificate Authentication` credential containing the server certificates.
+
+![server credentials](https://raw.githubusercontent.com/jenkinsci/docker-build-publish-plugin/master/credentials_server.png)
```

Comment:
I thought there was a reason for that, copying to the wiki page or rendering it somewhere else, not a big deal